api/v1alpha3/machinedeployment_types.go

```diff
@@ -174,15 +174,60 @@ type MachineDeploymentStatus struct {
 	// that still have not been created.
 	// +optional
 	UnavailableReplicas int32 `json:"unavailableReplicas,omitempty" protobuf:"varint,5,opt,name=unavailableReplicas"`
+
+	// Phase represents the current phase of a MachineDeployment (ScalingUp, ScalingDown, Running, Failed, or Unknown).
+	// +optional
+	Phase string `json:"phase,omitempty"`
 }
 
 // ANCHOR_END: MachineDeploymentStatus
 
+// MachineDeploymentPhase indicates the progress of the machine deployment
+type MachineDeploymentPhase string
+
+const (
+	// MachineDeploymentPhaseScalingUp indicates the MachineDeployment is scaling up.
+	MachineDeploymentPhaseScalingUp = MachineDeploymentPhase("ScalingUp")
+
+	// MachineDeploymentPhaseScalingDown indicates the MachineDeployment is scaling down.
+	MachineDeploymentPhaseScalingDown = MachineDeploymentPhase("ScalingDown")
+
+	// MachineDeploymentPhaseRunning indicates scaling has completed and all Machines are running.
+	MachineDeploymentPhaseRunning = MachineDeploymentPhase("Running")
+
+	// MachineDeploymentPhaseFailed indicates there was a problem scaling and user intervention might be required.
+	MachineDeploymentPhaseFailed = MachineDeploymentPhase("Failed")
+
+	// MachineDeploymentPhaseUnknown indicates the state of the MachineDeployment cannot be determined.
+	MachineDeploymentPhaseUnknown = MachineDeploymentPhase("")
```

Comment:
Should this align with the recent changes to Cluster/Machine phases and use `"Unknown"` here instead of `""`?

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/1532/files for reference

Reply:
makes sense to follow the same standard, will change it